### PR TITLE
Publish it to npm when creating a release

### PR DESCRIPTION
**Pull request by wiese**
I am not sure if this will work - there is documentation for different evolutions of github actions available which makes things rather blurry.

Todo
- [x] [add secret](https://github.com/wmde/vuex-helpers/settings/secrets) (npm token)

### CODE REVIEW

File: .github/workflows/npm-publish.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: https://registry.npmjs.org/
+      - run: npm ci
+      # ensures latest dist/ through build in prepublishOnly
+      - run: npm publish
+        env:
+          NODE_AUTH_TOKEN: ${{secrets.npm_token}}
```
Comment (manicki) [inline .github/workflows/npm-publish.yml:23]:
```suggestion
          NODE_AUTH_TOKEN: npm_token
```

Comment (manicki) [reply .github/workflows/npm-publish.yml:23]: added token and named in a very surprising way like suggested above

Comment (manicki) [reply .github/workflows/npm-publish.yml:23]: unless I am suggesting crap and actually the code above is fine and will be processed as intended by github

Comment (wiese) [reply .github/workflows/npm-publish.yml:23]: Is effectively a [copy from here](https://github.com/actions/starter-workflows/blob/master/ci/npm-publish.yml#L33) but I would not mind you being right.

Comment (manicki) [reply .github/workflows/npm-publish.yml:23]: ignore the suggestion please

**Review by manicki — APPROVED** [review]

**Pull request merged**